### ISSUE TITLE
Add basic NES emulator support to the main Hakchi install?

### ISSUE THREAD

**Issue by BagginsKQ** [ClusterM/hakchi2#158]
How difficult would it be to include a very basic retroarch mod with only cores for Fceux or one of hte other emulators or both as needed for the games not compatible with the base NIntendo Mini?

So that Hakchi2 program can install near "complete" nes compatiblity from the getgo, but without all the bloat from other non NES console emulator mods. So NES and only NES?

**Comment by BagginsKQ** [comment]
Also the way the smart system works does it also switch the games that 'work' but glitched garbled material to work on a better emulator?

**Comment by bfluryx1234** [comment]
Depending on when you installed Retroarch it has changed a little.  It used to be installed as one package with all the "bloat" of the other systems.  Now the core is seperate modification as you described, and even emulator is its own modification as well.  Meaning you can in effect accomplish this goal, although you still have to install the base Retroarch mod.

**Comment by abijanfar** [comment]
you can install RetroArch mod this comes with fceumm a much better emulator then the once that's in the box. Now you can include the global command arg "--retroarch" and all NES emulation will use the fceumm emulator.

**Comment by BagginsKQ** [comment]
Well I meant even without the bloat version it might be handy to include the mod directly in the hackchi download (still use the handy checkbox option though). It took me a while to figure out where each of the bits went, the instructions were a tad hard to follow. I finally figured it out and ended up placing the extra bloat files into a 'extras' folder until I actually choose to use them or not.

**Comment by ClusterM** [comment]
It's not good idea to put everything in one release.
